### DISCUSSION TITLE
Page builder

### DISCUSSION THREAD

**Discussion by nobears-maarten** [lucky-media/landtamic#2] [Q&A] [answered]
Can you guys show me the page builder you've created to manage all blocks and create pages?

**Comment by lokmanm** [comment]
Hi @pitcher-maarten, 

We are working on making a video tutorial that will showcase the functionality and how everything works. 

In the meantime if you don't mind I will drop the `yaml` file we use for page building. [Pages](https://github.com/lucky-media/landtamic/blob/main/resources/blueprints/collections/pages/pages.yaml)

Thanks

**Reply by nobears-maarten** [reply]
Thanks @lokmanm is there any way I can see the starter-kit CP myself without purchasing?

**Reply by lokmanm** [reply]
> Thanks @lokmanm is there any way I can see the starter-kit CP myself without purchasing?

I am afraid that is not possible for now but it's a good idea for the future. Maybe we can setup a demo account and just refresh the state every 3 hours or so. 

I can tag you once we publish the video we will go through all the features and I think you will have a better idea of what you get. 

Let me know if you have any other ideas we will gladly look over.

**Reply by nobears-maarten** [reply]
Perfect! Please, keep me updated about the video ;)

**Reply by lokmanm** [reply]
@pitcher-maarten no problem at all. Thanks

**Reply by lokmanm** [reply, marked as answer]
@pitcher-maarten as promised here is the [video](https://www.youtube.com/watch?v=XBa6MWjCZCs) showcasing all the features.
